### PR TITLE
Freshens memes

### PR DESCRIPTION
**Pull request by Ziiro**
:cl: 
add: brain damage lines
/:cl:

I'm not funny this pr is just here because I'm an asshole

### CODE REVIEW

**Comment by Cactus2388** [comment]
Remove the ;REEEE line so we can tell who's brain damaged and who's just being a LRP shitter please. We have enough of it anyways.

**Comment by Ziiro** [comment]
hang on I got a thing for that

**Comment by Ziiro** [comment]
```
if(findtext(message, "REEE"))
	usr.adjustBrainLoss(10)
	to_chat(usr, "<span class='danger'>Your high pitched screeching has given you brain damage.</span>")```

File: code/modules/mob/mob_helpers.dm [hunk 1]
```diff
@@ -196,6 +196,7 @@
 	message = replacetext(message, "space", "spess")
 	message = replacetext(message, "carp", "crap")
 	message = replacetext(message, "reason", "raisin")
+	message = replacetext(message, "please", "pwease")
```
Comment (FlattestGuitar) [inline code/modules/mob/mob_helpers.dm]: pweasw halp me uwu

Comment (Ziiro) [reply code/modules/mob/mob_helpers.dm]: I'll make periods filter to uwu

**Comment by Ziiro** [comment]
I'll legitimately revisit this later but for now this has given me an idea

**Pull request closed without merging**